app/src/main/res/layout/fragment_search_speakers.xml

```diff
@@ -16,6 +16,8 @@
             android:id="@+id/search_session_recycler"
             android:layout_width="0dp"
             android:layout_height="0dp"
+            android:layout_marginTop="16dp"
+            android:layout_marginBottom="16dp"
```

Comment:
Its not good to set margin to RecyclerView.
because it makes scroll behavior strange.
What do you think?

Reply:
I see!
thanks for pointing out! 🙇